config/routes.rb

```diff
@@ -5,5 +5,10 @@
     resources :posts
   end
 
+  resources :posts do
+    resources :comments, only: [:create]
```

Comment:
- [ ] Kindly ensure that there is a pair of routes in the `config/routes.rb` file for creating comments like below:

```ruby
   POST   /posts/:post_id/comments(.:format)     comments#create
   GET   /posts/:post_id/comments/new(.:format)     comments#new
```

Reply:
Done, Thank you!